routes/index.js

```diff
@@ -15,9 +15,26 @@ router.get('/', function(req, res) {
 
 /* GET /notebooks - list of notebooks */
 router.get('/notebooks', function(req, res) {
-    res.render('index', {
-        title: 'Notebooks'
-    });
+    nbstore.getNotebooks().then(
+        function success(notebooks) {
+            //if notebook index exists redirect to it immediately
+            var notebookIndex = "index.ipynb"
+            if(notebooks.indexOf(notebookIndex) > -1) {
+                res.redirect('/notebooks/' + notebookIndex);
+            }
```

Comment:
Also, a `redirect` will change the client's URL bar. Might be better to just serve the file at `/` instead. Is that easy to do here?